superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -18,6 +19,48 @@ const ROW_LIMIT_OPTIONS = [10, 50, 100, 250, 500, 1000, 5000, 10000, 50000];
 
 const SERIES_LIMITS = [0, 5, 10, 25, 50, 100, 500];
 
+const metricRenderer = m => (
```

Comment:
ideally components shouldn't live in a stores file. these should probably be their own component files so they can be used in other places as needed.